boot/bootutil/include/bootutil/crypto/sha256.h

```diff
@@ -27,6 +28,9 @@
 
 #if defined(MCUBOOT_USE_MBED_TLS)
     #include <mbedtls/sha256.h>
+#if defined (MBED_TLS_VERSION_3_0_0)
```

Comment:
Also, there shouldn't be a space between `defined` and the `(`.

Reply:
Done